### ISSUE TITLE
System Container shows broken images when adding

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#23088]
The System Container always tries to show an image for any piece of content.  When you add a new piece of content to the system container, even if the content does not have an image, it shows it as a broken image.  Additionally, when you refresh, it works, but the image is not thumbnailed and can take up the whole screen.

https://user-images.githubusercontent.com/934364/193845196-aec88091-8b27-4f46-a830-02ec03ad6875.mov

Bottom line - the system container needs some logic/styling to display in a card like fashion, with nice fallbacks if it can't find titles, images.

### Acceptance criteria
1. Image should show ONLY when added
2. No image we should nothing
3. If the image if big we have to constrain to max the contentlet width

**Comment by fmontes** [comment]
Needs groom and AC

**Comment by fmontes** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Maintenance-61770c654506a300167b30f4/issues/dotCMS/core/23088?planning-poker) with Zenhub @hmoreras @jcastro-dotcms @jdotcms @rjvelazco

**Comment by jcastro-dotcms** [comment]
## Internal QA: PASSED
Contentlets in the page are showing up in a card-like style:

<img width="1052" alt="test" src="https://user-images.githubusercontent.com/8779585/198096935-88928dd3-2d4e-4518-a7e7-d151c20c1c22.png">

**Comment by jcastro-dotcms** [comment]
After discussing new findings with @fmontes , we noticed that using the `$dotContentMap` solves a lot of the difficulties in the code fix. however, we noticed that the `title` property in `master` is returning the Identifier instead of the next Text field value, as it is in `release-22.10` right now. I'll look into it.

**Comment by fmontes** [comment]
Internal QA, test binary, dotasst and images field.

<img width="994" alt="image" src="https://user-images.githubusercontent.com/751424/200047497-d5bee9d6-ad8f-4546-8f74-e7306e709406.png">

**Comment by bryanboza** [comment]
Fixed, tested on release-22.11 // Docker // FF. Now we are showing the thumbnails as we show in the last comment
